README.md

```diff
@@ -182,5 +182,13 @@ New to open source/free software? Here is a selection of issues we've made **esp
 
 [Here](https://publiclab.org/notes/warren/11-22-2017/use-git-and-github-to-contribute-and-improve-public-lab-software) is a link to our Git workflow.
 
-## Let the code be with you. :heart:
+## Let the code be with you. 
 ### Happy opensourcing. :smile:
+
+<hr>
+
+<center>
+
+#### [Platforms that :heart: OSS](./doc/SUPPORTERS.md)
```

Comment:
```suggestion
#### [Platforms that :heart: FOSS](./doc/SUPPORTERS.md)
```

Reply:
Just wondering about `FOSS` because I have never used it and just googled it:

I often see on organizations websites:  `<company_name> loves open source` to indicate that their typically paid packages are free for OS projects - see [browserStack page](https://www.browserstack.com/open-source?ref=footer) for ex. 

but if you look at the actual qualifications on the same page linked above: 
<blockquote>
Qualify: Make your code public; add an open source license.
</blockquote>

1) If your code is public, does this mean that you have a `FOSS` project rather than just an `OSS` project? Are there cases where code is public but the project is not `FOSS`? 
2) As I mentioned above and linked with browserStack, for ex, I do not see `FOSS` being used very often. Is it because it is a narrower category of projects or bc the distinction is overlooked often enough that `OSS` in many contexts just implies `FOSS`? 
3) Anyway should I start referring to PL projects as `FOSS` not `OSS`? 
4) The platforms linked don't ever mention `FOSS` but depending on your answers to 1, 2, 3 (sorry!!) I will commit this.

Thank you looking forward to being knowledgeable on this

Reply:
First of all, I want to mention that I myself is not that knowledgeable to review the usage of this term. It's really a topic of discussion - https://opensource.com/article/17/11/open-source-or-free-software

> If your code is public, does this mean that you have a FOSS project rather than just an OSS project? Are there cases where code is public but the project is not FOSS?

According to https://opensource.com/article/18/3/what-open-source-programming - Let's be clear up front about something: Just being on GitHub in a public repo does not make your code open source.

>  Are there cases where code is public but the project is not FOSS?

Every other repo which is unlicensed but jokes apart, I think it depends on the license used by an org for the project. 

> As I mentioned above and linked with browserStack, for ex, I do not see FOSS being used very often. Is it because it is a narrower category of projects or bc the distinction is overlooked often enough that OSS in many contexts just implies FOSS?

Like, written in the above article, many people refer to FOSS as a neutral term, is it appropriate? No idea at all!!  

> Anyway should I start referring to PL projects as FOSS not OSS?

I myself use OSS at most places but it's good question for @jywarren. Also, @jywarren if you can clear the difference between OSS and FOSS and which one should be preferred then it would be awesome!!! Thanks! 

> The platforms linked don't ever mention FOSS but depending on your answers to 1, 2, 3 (sorry!!) I will commit this.

Don't bother about committing this as it was a suggestion(although, a bad one) but I like this discussion :sweat_smile:.  Also, I checked every platform's website and they do provide us free services but i am not clear about Zenhub. So, I am fine with using OSS, but would like to know, if ZenHub free for Open Source projects?

Thanks @sashadev-sky !!!

Reply:
@gauravano Yeah Zenhub is free for Open Source projects! I will go on a little reading binge and engage back in this discussion. Very curious about this whole thing.

So do you think we can merge this? but leave this part open so @jywarren can chime in

Reply:
Woah reading up from https://www.gnu.org/philosophy/free-sw.html, which is a link within a link you shared, it's a big ethical argument surrounding whether or not the user has full **control** over the software. 
> A program is free software if it gives users adequately all of these freedoms. Otherwise, it is nonfree. While we can distinguish various nonfree distribution schemes in terms of how far they fall short of being free, we consider them all equally unethical.

`FOSS` is basically similar to `OSS`, and `OSS` is often free. But when using the term `FOSS`, you are including connotations about the philosophical stand that software denying the **4  freedoms** is **unethical**. To me, it almost seems like `FOSS` is an "activist" word . Yes, it has some technical differences between `OSS` (As you mentioned, too, the licenses),  but there is really an underlying philosophy you are conveying. 

Do you think you would agree with this interpretation? I stated it like a fact but it is really just my own conclusions that I drew
@gauravano

Reply:
This conversation is so interesting!

In general "Open Source" licenses are also Free Software licenses because the "Open Source Definition" basically contains the Free Software criteria (four freedoms) - except very rare cases.

I _personally_ ascribe to the Free Software spirit and do my best to avoid non-freedom respecting technologies and to give others this perspective with a positive message that it is possible for technology to be liberating for the self and society.

In context, the README already has a liberal mix of both terms so I think it's fine!

:palm_tree: :parasol_on_ground: :statue_of_liberty: :heart:

Reply:
Thanks @sashadev-sky, I skipped reading the 4 freedoms earlier but gone through the whole doc(link which you pasted) and also reading @icarito's gave me idea!

Thanks @icarito for insights, let's keep it `OSS` then :smile: